packages/flutter_tools/lib/src/runner/target_devices.dart

```diff
@@ -21,6 +21,8 @@ const String _noAttachedCheckForWirelessMessage = 'No devices found yet. Checkin
 const String _noDevicesFoundMessage = 'No devices found.';
 const String _noWirelessDevicesFoundMessage = 'No wireless devices were found.';
 const String _wirelesslyConnectedDevicesMessage = 'Wirelessly connected devices:';
+String flutterSpecifiedDeviceDevModeDisabled(String deviceName) => 'To use '
+    "'$deviceName' for development, enable Developer Mode in Settings → Privacy & Security.";
```

Comment:
nit: The const strings and the function string messages are separated and in alphabetical order here. Could be nice to have it fit in with that